nextcord/interactions.py

```diff
@@ -1252,3 +1257,44 @@ class InteractionMessage(_InteractionMessageMixin, Message):
     """
 
     pass
+
+
+class MessageInteraction:
```

Comment:
This class should probably be added to api.rst so it shows up in the docs

Reply:
Should be added, please check if anything is wrong when you can 🤍